src/main/scala/fasta.scala

```diff
@@ -88,7 +109,7 @@ case object fasta {
       getH: AnyApp1At[findS[AnyDenotation { type Tpe = header.type }],RV] { type Y = header.type := header.Raw },
       getS: AnyApp1At[findS[AnyDenotation { type Tpe = sequence.type }],RV] { type Y = sequence.type := sequence.Raw }
     )
-    : String = s"${fa.head.value.asString}\n${fa.tail.head.value.lines.grouped(70).mkString("\n")}\n"
+    : String = s"${fa.head.value.asString}\n${fa.tail.head.value.lines.grouped(70).mkString("\n")}"
```

Comment:
A bit more civilized way to do it:
```scala
  (fa.head.value.asString :: fa.tail.head.value.lines.grouped(70)).mkString("\n")
```

Also, if it returns `String`, it should be something like `asString`. And implicits are not needed.